### DISCUSSION TITLE
No luck installing. Please help!

### DISCUSSION THREAD

**Discussion by scott4382** [Xaymar/obs-stream-effects#995] [Q&A] [answered]
I've tried installing both the current production plugin on my MacBook M1 Pro running OBS 28.1.2. I followed the installation guide to the letter and got the successful installation message, but then nothing shows up in the my plugin folder or in OBS itself. I even tried uninstalling and installing the current beta, but that didn't work either. I'm new to both OBS and StreamFX but really hope to get this working. Any ideas?

**Comment by skeletonbow** [comment, marked as answer]
> I've tried installing both the current production plugin on my MacBook M1 Pro running OBS 28.1.2. I followed the installation guide to the letter and got the successful installation message, but then nothing shows up in the my plugin folder or in OBS itself. I even tried uninstalling and installing the current beta, but that didn't work either. I'm new to both OBS and StreamFX but really hope to get this working. Any ideas?

Your OBS release is not compatible, be sure to review the System Requirements carefully which list the required OBS release for each release of the plugin.  It currently requires OBS 29.0.0beta2 to use the latest build, and the latest stable release of the plugin only supports OBS 27.2.4.

**Comment by ghost** [comment]
> I've tried installing both the current production plugin on my MacBook M1 Pro running OBS 28.1.2. I followed the installation guide to the letter and got the successful installation message, but then nothing shows up in the my plugin folder or in OBS itself. I even tried uninstalling and installing the current beta, but that didn't work either. I'm new to both OBS and StreamFX but really hope to get this working. Any ideas?

Streamfx is working on beta version 29 of OBS. In version 28 it doesn't work
